i2csoft/i2csoft.go

```diff
@@ -278,3 +280,8 @@ func (i2c *I2C) WriteRegister(address uint8, register uint8, data []byte) error
 func (i2c *I2C) ReadRegister(address uint8, register uint8, data []byte) error {
 	return i2c.Tx(uint16(address), []byte{register}, data)
 }
+
+// wait waits for half the time of the SCL operation interval.
+func (i2c *I2C) wait() {
+	delay.Sleep(50_000) // half of a 100kHz cycle (50µs)
```

Comment:
Why not `50 * time.Microsecond` as an argument?

Reply:
Yes you are right, I've fixed it.